lib/wallet/wallet.js

```diff
@@ -1117,8 +1117,14 @@ class Wallet extends EventEmitter {
       throw new Error('Account not found.');
 
     let rate = options.rate;
-    if (rate == null)
+    if (rate == null) {
+      // No fee rate specified, check for confirmation target to estimate fee
```

Comment:
I'm pretty sure this isn't doing anything.

Reply:
Right , needs to be this?

`rate = await this.wdb.estimateFee(blocks);`

Reply:
Yeah, I don't think any changes are needed here.